resources/celery_tasks.py

```diff
@@ -0,0 +1,30 @@
+import requests
+import json
+
+from .helper import Helper
+
+"""
+- ERROR 403 v1/celery_tasks/last/ 
+- ERROR 404 v1/celery_tasks/last/{org_pk}/
+
+"""
+
+class Celery_tasks(Helper):
+    def __init__(self, base_url, org_pk, teams_pk, access_token, _csrf_token, headers, pagination):
+        super().__init__(base_url, org_pk, teams_pk, access_token, _csrf_token, headers, pagination)
+
+    # TODO GET on /api/v1/celery_tasks/last/
```

Comment:
I suggest to write
TODO(who writed) blablabla

Then, if we have any problem to understand what is needed here, we understand who we can to adress our question.

Reply:
Edit : Added to proposed changes > [Back-end conventions](https://docs.google.com/document/d/1c9a7eqfc258lZme9TcMJpr3crO-v2nlCq9jAHQmxDTc/edit#heading=h.wnbj3sta1a8y)